hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -1999,6 +2032,47 @@ protected void testTakedown() throws IOException {
     }
   }
 
+  /*
+  Send random reads against fake regions inserted by MetaWriteTest
+   */
+  static class MetaRandomReadTest extends MetaTest {
+    private Random rd = new Random();
```

Comment:
Since we do not pre set a seed here, please just use ThreadLocalRandom instead.

Reply:
Changed to existing random instance that is initialized by current system time.